### ISSUE TITLE
@:default values in JsonSchemaWriter

### ISSUE THREAD

**Issue by Gama11** [elnabo/json2object#56]
```haxe
import json2object.utils.JsonSchemaWriter;

class Main {
	static function main() {
		trace(new JsonSchemaWriter<Data>("\t").schema);
	}
}

typedef Data = {
	@:default(10)
	var foo:Int;
	@:default("hello world")
	var bar:String;
}
```

Generates:

```json
{
        "$schema": "http://json-schema.org/draft-07/schema#",
        "$ref": "#/definitions/Data",
        "definitions": {
                "Data": {
                        "additionalProperties": false,
                        "properties": {
                                "bar": {
                                        "type": "string"
                                },
                                "foo": {
                                        "type": "integer"
                                }
                        },
                        "required": [
                                "bar",
                                "foo"
                        ],
                        "type": "object"
                }
        }
}
```

What it should be generated as (important to include the `default` field for code completion):

```json
{
	"$schema": "http://json-schema.org/draft-07/schema#",
	"$ref": "#/definitions/Data",
	"definitions": {
		"Data": {
			"additionalProperties": false,
			"properties": {
				"bar": {
					"type": "string",
					"default": "hello world"
				},
				"foo": {
					"type": "integer",
					"default": 10
				}
			},
			"required": [
				"bar",
				"foo"
			],
			"type": "object"
		}
	}
}
```

**Comment by elnabo** [comment]
This is supported since c357b2c443a89a099ff83bdae3b49f40359969ec.

**Comment by Gama11** [comment]
Hm, doesn't seem to compile anymore.

```
json2object\src\json2object/utils/schema/DataBuilder.hx:268: characters 77-83 : Unknown identifier : putils
json2object\src\json2object/utils/schema/DataBuilder.hx:268: characters 77-83 : For optional function argument 'putils'
```

**Comment by elnabo** [comment]
Ok, I see the problem.

It's a problem with `@:default(auto)` and a bad copy/paste.

I should be able to fix it later today.

**Comment by elnabo** [comment]
Should be fixed by 30cae7c7afa4a8b447f577cf4d73535476d3dded
